public/index.html

```diff
@@ -3,6 +3,8 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="preconnect" href="https://fonts.gstatic.com">
```

Comment:
crossorigin ?

Reply:
I have moved font loading to index.css and save fonts to src/fonts folder to prevent crossorigin